src/auro-popover.js

```diff
@@ -89,28 +78,28 @@ class AuroPopover extends LitElement {
     handleHide = () => {
       this.toggleHide();
     },
-    handleTabWhenFocusOnTrigger = (event) => {
-      if (event.key.toLowerCase() === 'tab') {
-        this.toggleHide();
-      }
-    };
+    handleKeyboardWhenFocusOnTrigger = (event) => {
+      const key = event.key.toLowerCase();
 
-    if (!this.sticky) {
-      this.trigger.addEventListener('touchstart', handleShow);
-    }
+      if (this.isPopoverVisible) {
+        if (key === 'tab' || key === 'escape') {
```

Comment:
This should fire when the trigger has lost focus

Reply:
I don't think changing focus is the only case in which the content should be dismissed. These key presses allow the user finer control.